check/reference/readme.md

```diff
@@ -14,4 +14,4 @@ layout:
+<table data-column-title-hidden data-view="cards"><thead><tr><th></th><th data-hidden></th><th data-hidden></th><th data-hidden data-card-cover data-type="files"></th><th data-hidden data-type="checkbox"></th><th data-hidden data-card-target data-type="content-ref"></th></tr></thead><tbody><tr><td>Trunk YAML</td><td></td><td></td><td><a href="../../.gitbook/assets/TrunkYaml (1).svg">TrunkYaml (1).svg</a></td><td>false</td><td><a href="trunk-yaml.md">trunk-yaml.md</a></td></tr><tr><td>User YAML</td><td></td><td></td><td><a href="../../.gitbook/assets/UserYaml.svg">UserYaml.svg</a></td><td>false</td><td><a href="user-yaml.md">user-yaml.md</a></td></tr><tr><td>Integration for Slack</td><td></td><td></td><td><a href="../../.gitbook/assets/Slack (1).svg">Slack (1).svg</a></td><td>false</td><td><a href="../../administration/integration-for-slack.md">integration-for-slack.md</a></td></tr><tr><td>VS Code</td><td></td><td></td><td><a href="../../.gitbook/assets/VSCode (1).svg">VSCode (1).svg</a></td><td>false</td><td><a href="../ide-integration/vs-code.md">vs-code.md</a></td></tr><tr><td>Neovim</td><td></td><td></td><td><a href="../../.gitbook/assets/neo_vim.svg">neo_vim.svg</a></td><td>false</td><td><a href="../ide-integration/neovim-plugin.md">neovim-plugin.md</a></td></tr><tr><td>Usage Data</td><td></td><td></td><td><a href="../../.gitbook/assets/Usage Data.svg">Usage Data.svg</a></td><td>false</td><td><a href="usage-data.md">usage-data.md</a></td></tr><tr><td>Components</td><td></td><td></td><td><a href="../../.gitbook/assets/Components (1).svg">Components (1).svg</a></td><td>false</td><td><a href="components.md">components.md</a></td></tr><tr><td>Glob Path Pattern</td><td></td><td></td><td><a href="../../.gitbook/assets/Glob.svg">Glob.svg</a></td><td>false</td><td><a href="glob-path-pattern.md">glob-path-pattern.md</a></td></tr></tbody></table>
```

Comment:
I'm not sure how this works in Gitbooks but can we double check that we don't have the `(1)` in the UI?